QUEST_LV_0100.tsv

```diff
@@ -1424,7 +1424,7 @@ QUEST_LV_0100_20150317_001423	The destruction of the capital was just the beginn
 QUEST_LV_0100_20150317_001424	Monstrous plants sprouted up all over and began devouring people.{nl} Everyone started running, but before I knew it I was the only person left.{nl}
 QUEST_LV_0100_20150317_001425	Enough... Don't ask any more questions. {nl} I don't want to think about it anymore.
 QUEST_LV_0100_20150317_001426	From here, it's Zachariel's Mausoleum excavation site.{nl}You can freely enter, but please be careful not to disturb the research.
-QUEST_LV_0100_20150317_001427	Niels is the one who is in charge of this place.{nl}If you need to ask something, go find Niles at the researchers' camp.
+QUEST_LV_0100_20150317_001427	$Niels is the one who is in charge of this place.{nl}If you need to ask something, go find Niels at the researchers' camp.
```

Comment:
I'm in the midst of checking the Revelation story of the Royal Mausoleum.
Check my changes from this commit: https://github.com/Sunaries/EnglishTranslation/commit/f2891bcd72fa0e77dbd2411ffffc278b8530504e

Basically, I have also changed this sentence. However, my edits don't have a $ because I don't consider everything just final yet. There are Korean lines hidden in the first couple of Revelation stories, so this allows us to check it again.